### ISSUE TITLE
Hide Dash X Incompatible With Gnome 40

### ISSUE THREAD

**Issue by LunaGNUisance** [Edenhofer/Minimalism-Gnome-Shell#14]
This looked like a handy extension, but unfortunately, I can't use it on GNOME 40. I'd appreciate if you spent the time updating the extension to work with GNOME 40. Thanks!

**Comment by Edenhofer** [comment]
I am sorry but as of now these extensions are kind of stale. I stopped using Gnome a while ago and the development was stopped. Furthermore, I have no time for reading up on the new extension mechanisms in Gnome 40.
Having said that, I would be glad to review and test any patches that are sent my way!

**Comment by LunaGNUisance** [comment]
Fair enough; thank you for the quick response. If I get interested enough, I might do the reading and attempt to port it over myself, but I expect that won't be for some time, if at all. Thank you for doing the work originally and sharing it with the public, though, as I'll have something to base it off, should I go that route!
